client/state/themes/reducer.js

```diff
@@ -459,6 +462,28 @@ export const themeFilters = withSchemaValidation( themeFiltersSchema, ( state =
 	return state;
 } );
 
+/**
+ * Returns updated state for recommended themes after
+ * corresponding actions have been dispatched.
+ *
+ * @param  {object} state  Current state
+ * @param  {object} action Action payload
+ * @returns {object}        Updated state
+ */
+const recommendedThemes = ( state = { isLoading: false, themes: [] }, action ) => {
```

Comment:
I think we usually try to add tests for new parts of the state. You can probably find examples by looking at other reducers :)

Reply:
Great suggestion.  I have added some basic tests for the actions and reducers.